src/content/blog/2024-12-17-rafiki-tigerbeetle-integration.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "Balancing the Ledger: Rafiki TigerBeetle Integration"
+description: "How TigerBeetle Supercharges Rafiki’s Financial Core"
+date: 2024-12-17
+slug: rafiki-tigerbeetle-integration
+authors:
+  - Jason Bruwer
+author_urls:
+  - https://www.linkedin.com/in/jason-bruwer-8110766/
+tags:
+  - Interledger
+  - Interledger Protocol
+  - Interledger Stack
+  - Interledger Foundation
+  - Open Payments
+  - Rafiki
+  - TigerBeetle
+---
+
+Rafiki is an open-source platform that enables Account Servicing Entities (ASEs) like banks and digital wallet providers to integrate Interledger Protocol (ILP) functionality into their systems.
+
+A critical component of Rafiki’s architecture is its accounting database, which manages financial transactions and account balances. To enhance performance and reliability, Rafiki offers the option to use TigerBeetle, a specialized financial accounting database, as its accounting backend.
+
+![Balancing the Ledger](/developers/img/blog/2024-12-17/balancing.png)
```

Comment:
what do you think of instead having something like [Rafiki Logo x Tigerbeetle logo]?
I can make it quick if you'd like

Reply:
Yes please.